### ISSUE TITLE
Facebook chat / Messenger support

### ISSUE THREAD

**Issue by MrSaints** [papercups-io/papercups#141]
Similar to https://github.com/papercups-io/papercups/issues/94, but for Facebook.

This is particularly useful if you are operating a Facebook page because you can respond to support requests over your page's messenger.

**Comment by reichert621** [comment]
@MrSaints sounds interesting! how would this work if someone was messaging you through your website and didn't have a facebook account? (my assumption is that facebook messenger can only accept messages from people who have facebook authenticated, right?)

**Comment by MrSaints** [comment]
@reichert621 Oh, I was thinking more in terms of simply initiating the conversation from a Facebook chat :) Apologies if this is sort of a non-goal for this project.

But I can foresee a more complex integration involving someone being signed into Facebook, and initiating it from an embeddable on the website too, though I'm not sure what the limitations are in terms of Facebook's API, and SDKs (https://developers.facebook.com/docs/messenger-platform/discovery/facebook-chat-plugin/).

**Comment by reichert621** [comment]
@MrSaints oh i see! so you mean if someone initiates a chat from your facebook page, the message can be passed through papercups?

**Comment by MrSaints** [comment]
@reichert621 Yup :) And responses will be passed back through to the ongoing direct message on Facebook.
